storage/engine/rocksdb/db.cc

```diff
@@ -115,6 +120,15 @@ DBStatus ToDBStatus(const rocksdb::Status& status) {
   return ToDBString(status.ToString());
 }
 
+DBStatus FmtStatus(const char *fmt, ...) {
```

Comment:
Is this function covered in our tests anywhere? I didn't see any cases where we test for a failure of MVCCComputeStats, but I didn't do an exhaustive search. Would be good to have tests to make sure we get the memory management right here.